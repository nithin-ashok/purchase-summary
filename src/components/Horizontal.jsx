import React from "react";

const Horizontal = ()=>(<div className="hr"></div>)

export default Horizontal;