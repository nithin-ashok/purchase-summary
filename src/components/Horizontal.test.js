import React from 'react';
import Horizontal from "./Horizontal";
import renderer from 'react-test-renderer';

it("renderes correctly", () => {
    const tree = renderer.
    create(<Horizontal />).toJSON();

    expect(tree).toMatchSnapshot();
});