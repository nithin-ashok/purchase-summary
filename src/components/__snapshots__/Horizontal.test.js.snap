// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderes correctly 1`] = `
<div
  className="hr"
/>
`;
